presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix rare (but practical) incorrect query result for queries involving timestamp,
+  when the time zone database embedded in the version of Joda-Time that Presto depends on
+  does not match the time zone database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when optimize_mixed_distinct_aggregation is enabled.
```

Comment:
what used to happen and what happens now?

Reply:
@sopel39 Karol, could you help answer Rebecca's question?